pages/api/formdata/[id].js

```diff
@@ -0,0 +1,53 @@
+import dbConnect from "../../../utils/dbConnect";
+import Cards from "../../../models/Cards";
+
+dbConnect();
+
+export default async (req, res) => {
+  const {
+    query: { id },
+    method,
+  } = req;
+
+  switch (method) {
+    case "GET":
+      try {
+        const formdata = await Cards.findById(id);
+        !Cards
+          ? res.status(400).json({ success: false })
+          : res.status(200).json({ success: true, data: formdata });
```

Comment:
If you don't use the return value of an expression, it's better to write a statement. In other words: You should use if / else instead of the ternary operator here.